src/test/java/org/betonquest/betonquest/api/bukkit/config/custom/multi/MultiSectionConfigurationModificationTest.java

```diff
@@ -95,7 +154,23 @@ void testMergedResult() {
             yamlConfig.set(entry.getKey(), entry.getValue());
```

Comment:
There is no need to copy all values from the merged config (`MultiConfiguration`) into a `YamlConfiguration` if no methods or features specific to that class are used.

With the above comment included a better version of this test would like this:
```java
    @Test
    void testMergedResult() {
        assertConfigurationEquals(configAll, config);
    }
```

I can't use suggest here as this part of the code wasn't yet changed in this PR.